docs/faq/nuclei.md

```diff
@@ -21,29 +21,29 @@
+	The Nuclei project is developed and maintained by [ProjectDiscovery](https://projectdiscovery.io/#/) team and is in active development (we generally release every other week).
 
 ??? tip "How can I support/contribute to this project? 💙"
 
-	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use / up-to-date nuclei templates.
+	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use, up-to-date nuclei templates.
 
-	If you found an interesting/unique security issue using nuclei and wanted to share the process walk-through with everyone in the form of a blog, we are happy to publish your guest blog at https://blog.projectdiscovery.io.
+	If you found an interesting/unique security issue using nuclei and want to share the process walk-through with everyone in the form of a blog, we are happy to publish your guest blog at https://blog.projectdiscovery.io.
 
 ??? warning "I found results with nuclei. When should I report it?"
 
-	**Wait for a moment**, after nuclei detected security issue, it's always advised to have a second look before reporting it. Here is a tip to confirm/validate the found matches.
+	**Wait a minute** -- after nuclei detected a security issue, it's always advised to have a second look before reporting it. Here's a tip to confirm/validate the found matches.
 
-	??? tip "How to validate nuclei results!"
+	??? tip "How do I validate nuclei results?"
 
-		Once nuclei finds a result, you have vulnerable ==URL== and ==template==, rerun the template with ==`-debug`== flag to inspect vulnerable response against expected matcher defined in the template. In this way, you can confirm the identified vulnerability.
+		Once nuclei finds a result, and you have vulnerable ==URL== and ==template==, rerun the template with ==`-debug`== flag to inspect the vulnerable response against expected matcher defined in the template. In this way, you can confirm the identified vulnerability.
 
 ??? warning "How much traffic does nuclei generate?"
 	
-	As default nuclei makes **1234** HTTP requests in total against single target upon running **all nuclei-templates** directory. This includes 801 nuclei templates from [v8.1.9](https://github.com/projectdiscovery/nuclei-templates/releases/tag/v8.1.9) release.
+	As default nuclei makes **1234** HTTP requests in total against single target upon running **all nuclei-templates** directory. This includes over 3500 nuclei templates from the [v9.0.2](https://github.com/projectdiscovery/nuclei-templates/releases/tag/v9.0.2) release.
```

Comment:
We might want to refrain ourselves from trying to give accurate numbers, because those numbers change often, and it would require periodical manual maintenance.

We could just rather says thousands and point to the releases page.